src/components/add-form/index.js

```diff
@@ -3,48 +3,52 @@ import RadioInput from '../radio-input';
 import './styles.css';
 
 const AddForm = ({ formState, handleForm }) => (
-  <>
-    <label htmlFor="productName">
+  <div className="addform-container">
+    <label htmlFor="productName" className="addform-container-label">
       Item name:
       <input
         className="input-item"
         id="productName"
         name="productName"
         type="text"
-        placeholder="Item"
+        maxLength="30"
+        placeholder="Write here the item to add..."
         value={formState.productName}
         onChange={(event) => handleForm(event)}
         required
       />
     </label>
     <fieldset className="time-frame" defaultValue="7">
-      <legend>How soon will you buy this again?</legend>
+      <legend className="time-frame-legend">
+        How soon will you buy this again?
+      </legend>
       <RadioInput
-        styles="time-frame-label"
         onChange={(event) => handleForm(event)}
         value="7"
         name="timeFrame"
         label="Soon"
+        className="time-frame-radio"
         required
         checked
       />
```

Comment:
What do you think about adding an `aria-label` to these radios? ❤️

Reply:
Good idea

Reply:
Done